pkg/sql/logictest/testdata/logic_test/collatedstring

```diff
@@ -507,3 +507,7 @@ query T
 SELECT s FROM nocase_strings2 WHERE s = ('bbb' COLLATE "en_US_u_ks_level2")
 ----
 Bbb
+
+statement ok
```

Comment:
nit: line 436 makes a `subtest nocase_strings`. so maybe we want a new subtest here?